### DISCUSSION TITLE
SOC-AI

### DISCUSSION THREAD

**Discussion by adriaanvermaak** [utmstack/UTMStack#747] [Q&A] [answered]
Hi Team,

Hope you are well.

Thank you for allowing me to test the UTM stack locally and absolutely loving it. Still learning how I can implement it and use it for my home business.

Question I have is we dont have the funding to utilise ChatGPT / Openai, is there a possibility to utilise a different provider like an open source one which alligns with openai like Ollama? as we have been playing around with a local but dont have the resources or funding to do either. 

hope you will be able to assist with how we can change the configuration to something like ollama as the Ai analyst or others.

thank you and looking forward to the response

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @adriaanvermaak, SOC-AI is a hardcoded integration, for now is not possible to change to another provider, we will take note of that for the future. Thanks for the feedback.
